### PR TITLE
Add generic errand machines and profile.

### PR DESCRIPTION
**Pull request by jmcarp**
n/t

### CODE REVIEW

File: cloud-config/cloud-config-base.yml [hunk 1]
```diff
@@ -15,6 +15,17 @@ vm_types:
       type: gp2
       size: 30000
       encrypted: true
+- name: errand_small
+  cloud_properties:
+    instance_type: t2.micro
```
Comment (cnelson) [inline cloud-config/cloud-config-base.yml]: t2.small?  We probably need at least 2GB of ram with all the addons we pack onto these machines

Comment (jmcarp) [inline cloud-config/cloud-config-base.yml]: ✅

**Comment by cnelson** [comment]
Should we add these new types and temporarily leave the old ones in place until we can update the logsearch and kubernetes deployments to use them?

**Comment by jmcarp** [comment]
I already updated the k8s feature branch, will pr logsearch now.

**Pull request merged**